tests/utils_tests/test_dateparse.py

```diff
@@ -80,6 +80,23 @@ def test_parse_python_format(self):
         for delta in timedeltas:
             self.assertEqual(parse_duration(format(delta)), delta)
 
+    def test_parse_postgres_format(self):
+        self.assertEqual(parse_duration('1 day'), timedelta(1))
```

Comment:
I'd use:
```
test_values = (
    ('1 day', timdelta(1)),
    ....
)
```
along with a loop and `self.subTest` to avoid the need to copy/paste `self.assertEqual(parse_duration` so much.